### PR TITLE
git-completion 2.12.0 (new formula)

### PR DESCRIPTION
**Pull request by lucidaeon**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/git-completion.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+class GitCompletion < Formula
+  desc "Bash completion for Git"
```
Comment (fxcoudert) [inline Formula/git-completion.rb]: Bash and Zsh

Comment (laurentr) [inline Formula/git-completion.rb]: Fixed!

**Comment by AraHaan** [comment]
Mind squashing the commits?

**Comment by laurentr** [comment]
@AraHaan Done.

**Comment by fxcoudert** [comment]
The `git`formula already has an option (on by default) to install bash and zsh completions (and also the `git-prompt.sh` script). So why provide these as a separate formula?

**Comment by fxcoudert** [comment]
The `conflicts_with` is a minimum, but it doesn't address the question: why would we want this? (Also, for zsh, system git comes with completion already!)

**Comment by laurentr** [comment]
For folks who don't have git installed from brew.

**Comment by fxcoudert** [comment]
macOS already has zsh completion installed in `/usr/share/zsh/5.2/functions/_git`. And for `bash`, if it is not already installed, then it needs to correspond to the system's `git` version, right?

**Comment by laurentr** [comment]
Yeah, makes sense.

**Pull request closed without merging**